docs/reference/expertfeatures.rst

```diff
@@ -0,0 +1,123 @@
+
+    If a full compile cannot be performed and handler code update is necessary,
+    it is possible to disable the version consistency check and to allow handler
+    code update in a manual export as follows:
+
+
+
+    .. danger::
+        Make sure the updated handler code is compatible with all resource
+        schemas it might have to process. Uploading incompatible handler code can lead
+        to very hard to diagnose issues.
+
+    .. admonition::
+        How-to
+
+        To allow handler code update during a partial compile, pass the
+        :command:`--allow-handler-code-update` option to the :command:`export` command.
+
+        .. code-block:: bash
+
+            inmanta -vv -X export --partial --allow-handler-code-update
+
+
+
+.. only:: iso
+
+    Lsm extension expert features
+    #############################
+
+    To access expert features of the lsm extension, toggle the
+    :inmanta.environment-settings:setting:`enable_lsm_expert_mode` environment setting. This will unlock
+    the features described below as long as the setting is still enabled. Make sure you disable it again when
+    you're done.
+
+    .. danger::
+        A service lifecycle describes how a service can/cannot evolve during its lifespan as a finite state
+        machine. An integral part of The LSM extension is to uphold these constraints by performing safety
+        check on each transition request.
+
+        Using LSM expert features explicitly bypass these safety checks, which means that incorrect usage
+        can lead to a broken desired state and break future compilations.
+
+        These features should never be used, unless you have a very good reason to, you fully understand
+        why the safety checks are there in the first place, and why it is ok to bypass them for your
+        specific use case.
+
+
+
+    Forcefully set a service in a state
+    -----------------------------------
+
+        .. danger::
+            This will forcefully set the service in the selected state, regardless of the limitations
+            imposed by the lifecycle. This will bypass all safety checks performed during regular
+            "update" state transfer.
+
+
+        .. admonition::
+            How-to
+
+            In the service instance view, click the :command:`Expert Actions` button, and then
+            select the new state from the :command:`Force state` section of the drop-down menu.
+
+
+    Forcefully destroy a service
+    ----------------------------
+
+        .. danger::
+            This will forcefully delete the service, regardless of the limitations
+            imposed by the lifecycle. This will bypass all safety checks performed during regular
+            "delete" state transfer. This will make the orchestrator "forget" the service and all the underlying
+            resources ever existed, and none of the on-deletion cleanup actions (described in the lifecycle, if any)
```

Comment:
that is not entirely true, the resources will only go on the next full compile or partial compile including this  instance.  So it is worse than this.